cohorts/plot.py

```diff
@@ -82,11 +137,16 @@ def fishers_exact_plot(data, condition1, condition2, ax=None, alternative="two-s
         x=condition1,
         y=condition2,
         ax=ax,
-        data=data
+        data=data,
+        **kwargs
     )
+
     count_table = pd.crosstab(data[condition1], data[condition2])
     print(count_table)
     oddsratio, pvalue = fisher_exact(count_table, alternative=alternative)
+    hide_negative_y_ticks(plot)
+    only_percentage_ticks(plot)
```

Comment:
`only_percentage_ticks` is only used for Fisher's exact right now. Does that answer your question?